util/src/test/java/tech/pegasys/artemis/util/config/ArtemisConfigurationTest.java

```diff
@@ -258,4 +262,60 @@ void loggingFileNamePatternShouldDefault() {
     final ArtemisConfiguration config = ArtemisConfiguration.fromString("");
     assertThat(config.getLoggingFileNamePattern()).isEqualTo("teku_%d{yyyy-MM-dd}.log");
   }
+
+  @Test
+  void validatorExternalSignerPublicKeysCanBeSet() {
+    final String publicKey1 =
+        "0xa99a76ed7796f7be22d5b7e85deeb7c5677e88e511e0b337618f8c4eb61349b4bf2d153f649f7b53359fe8b94a38e44c";
```

Comment:
These don't look like valid BLS keys - at 48 bytes long the start should be all 0's.  Note that `BLSPublicKey` doesn't actually parse the key until it's required (there are times the deposit contract can give us invalid keys). #1466 is dealing with the same issue so would be good to ensure we have a reusable `BLSPublicKey.isValid` method.

Reply:
Public keys are 48 bytes so this doesn't need changing